tests/functional/api/BackbeatServer.js

```diff
@@ -178,42 +184,33 @@ describe('Backbeat Server', () => {
 
         it('should get the right data for route: /_/metrics/backlog',
         done => {
-            statsClient.reportNewRequest(OPS, 5);
-            statsClient.reportNewRequest(BYTES, 7100);
-
             getRequest('/_/metrics/backlog', (err, res) => {
                 assert.ifError(err);
                 const key = Object.keys(res)[0];
-                assert.equal(res[key].results.count, 5);
-                assert.equal(res[key].results.size, 7.10);
+                assert.equal(res[key].results.count, 1275);
```

Comment:
in line 138: ```statsClient.reportNewRequest(OPS, 1725);```
should these 2 numbers be the same?

Reply:
Thanks for checking, in this case, I went ahead and selected fixed numbers for all tests to use, so the change here is accurate 👍